### DISCUSSION TITLE
Looking for team members for hackathon project on measuring the carbon impact of a website.

### DISCUSSION THREAD

**Discussion by heaversm** [Green-Software-Foundation/hack#98] [Matchmaking]
I am a developer at Mozilla. 

I have proposed the following project for Carbon Hack:

https://github.com/Green-Software-Foundation/hack/issues/97

I am attempting to source team members to work on this, both internally at Mozilla, and externally. While I have front end development and prototyping experience, and am well connected to folks within Mozilla who could help answer questions, I would love to find team members. Some ideas for who might be useful:

* Those who have experience or confidence using the Impact Framework or measuring software emissions
* Content writers
* People with Cloud Ops / Infrastructure experience
* People with data science / network monitoring skills
* Content writers and Designers
* UX / Front end folks
* Other?
 
If you are interested, feel free to chime in on this discussion with a description of your background and how you might like to participate.

**Comment by iamkennis** [comment]
I will love to join your team has a frontend developer, I have experience with Javascript/Typescript,ReactJs,Nodejs

**Comment by jeffreyzhangsd** [comment]
Hey Mike, I'm Jeffrey.

I'm interested! 

I'm a junior developer that just finished his first internship mostly working on a website, so hopefully that recent experience will be useful. I've used JavaScript, TypeScript, React, RESTful APIs, and am very open to learning more. To be more specific, I believe I can help with front end, documentation, and writing test cases. Anything else I would love to explore. 😀

**Reply by heaversm** [reply]
@jeffreyzhangsd  - great! Trying to figure out who from Mozilla will be participating, so I will be in touch with next steps, thanks for chiming in ( @jeffreyzhangsd  / @iamkennis )

**Reply by NAMRATA-WOKE** [reply]
@heaversm so glad to hear we might have teams from Mozilla!!

**Comment by NAMRATA-WOKE** [comment]
@heaversm you've got folks interested in joining your team!

**Comment by Bro-mar** [comment]
More than happy to help support across content writing and design!

Currently work in marketing where we create, design and publish reports across various sectors! So content and design would fall under strongsuits!

**Comment by heaversm** [comment]
Hi @jeffreyzhangsd , @Bro-mar  - feel free to register and list [our project github issue](https://github.com/Green-Software-Foundation/hack/issues/97) if you're still interested. Then add your name to [this spreadsheet](https://docs.google.com/spreadsheets/d/1vGK2-mbnNEhc6FDVqq3pAZYX6yTDIDsM5_nEDMrOS90/edit?userstoinvite=iretehamdani@gmail.com&sharingaction=manageaccess&role=writer#gid=0) so we can put together a contact list and try to coordinate a kickoff meeting.

[You can view this document to see what's been done so far. ](https://docs.google.com/document/d/1KTtqzWv345lxHaiX0OSJ5s-hGIti9qPEyszoMF3IieY/edit) - feel free to add comments / notes / get started testing out the framework and thinking about your own approach.
